Examples/Swift/ViewController.swift

```diff
@@ -148,6 +148,12 @@ class ViewController: UIViewController, MGLMapViewDelegate, NavigationViewContro
         navigationViewController.voiceController?.volume = 0.5
         navigationViewController.navigationDelegate = self
         
+        // Uncomment to apply custom styles
+//        styleForRegular().apply()
+//        styleForCompact().apply()
+//        styleForiPad().apply()
+//        styleForCarPlay().apply()
```

Comment:
I like the idea of a more direct use when we're talking specifically about UIViews and it crossed my mind a few times. However, a property like `maneuverViewHeight` will need some restrictions like MinMax(50, 200) to make sure the subviews fit inside.

An advanced user would probably already accomplish this by exposing a couple of private properties publicly and style anything using the appearance proxy but the intention here was also to keep it clean and simple. Changing the tint color for example updates `NavigationMapView`, `ProgressBar` and `Button`. Styling these w/o these helper methods would quickly become bloated.

```swift
NavigationMapView.appearance(for: traitCollection).tintColor = color
ProgressBar.appearance(for: traitCollection).backgroundColor = color
Button.appearance(for: traitCollection).tintColor = color
```
instead of
```swift
style.tintColor = color
```
excluding the initial style setup ofc.

Also, UIAppearance works on classes but not instances. We could expose ManeuverView but not as a property on NavigationViewController.